test/e2e/konnect_test.go

```diff
@@ -190,28 +192,113 @@ func createTestRuntimeGroup(ctx context.Context, t *testing.T) string {
 			req.Header.Set("Authorization", "Bearer "+konnectAccessToken)
 			return nil
 		}),
-		rg.WithHTTPClient(helpers.RetryableHTTPClient(helpers.DefaultHTTPClient())),
 	)
 	require.NoError(t, err)
+	createTimeStampStr := fmt.Sprintf("%d", time.Now().Unix())
+	createTestUUID := uuid.NewString()
+
+	var rgID uuid.UUID
+
+	createRgErr := retry.Do(func() error {
+		rgName := uuid.NewString()
+		createRgResp, err := rgClient.CreateRuntimeGroupWithResponse(ctx, rg.CreateRuntimeGroupRequest{
+			Description: lo.ToPtr("This is a description"),
```

Comment:
We might want to use the description to add a name of the test running here via `t.Name`. OTOH this is the only test that uses it so it would be better to include something like GITHUB CI run ID if present in the env.

We could use `GITHUB_SERVER_URL`, `GITHUB_SERVER_URL` and `GITHUB_RUN_ID` from https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables

This way if we encounter any errors and we're looking at the problematic RG in konnect we can easily link those 2 (RG with E2E run).

Reply:
done.